datasets/ru/rupep/crawler.py

```diff
@@ -322,17 +559,26 @@ def crawl_company(context: Context, data: Dict[str, Any]):
         "other_owners",
         "other_managers",
         "other_recipient",
+        "related_persons",
     ]
     context.audit_data(data, ignore=ignore)
-    # print(entity.to_dict())
     context.emit(entity)
```

Comment:
don't emit company here so we stop emitting government bodies unless we have some other relationship to them (Which we should probably get rid of at some point)